src/main/java/org/apache/sling/jcr/contentloader/internal/ContentLoaderWebConsolePlugin.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.sling.jcr.contentloader.internal;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Map;
+import java.util.Spliterator;
+import java.util.Spliterators;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import javax.jcr.RepositoryException;
+import javax.jcr.Session;
+import javax.servlet.GenericServlet;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+
+import org.apache.sling.api.request.ResponseUtil;
+import org.apache.sling.jcr.api.SlingRepository;
+import org.apache.sling.jcr.contentloader.PathEntry;
+import org.osgi.framework.Bundle;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.Constants;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+
+@Component(service=javax.servlet.Servlet.class,
+    property = {
+        Constants.SERVICE_VENDOR + "=The Apache Software Foundation",
+        Constants.SERVICE_DESCRIPTION + "=Apache Sling JCR Content Loader Web Console Plugin",
+        "felix.webconsole.label=" + ContentLoaderWebConsolePlugin.LABEL,
+        "felix.webconsole.title=JCR Content Loader",
+        "felix.webconsole.category=Sling"
+    })
+@SuppressWarnings("serial")
+public class ContentLoaderWebConsolePlugin extends GenericServlet {
```

Comment:
i've deployed this in a local AEMaaCS SDK instance and was not able to see the console.
do we have to extend AbstractWebConsolePlugin?

Reply:
I just tried this as well and there the component "org.apache.sling.jcr.contentloader.internal.BundleContentLoaderListener" is unsatisfied.
![Screenshot 2021-12-10 at 17 32 47](https://user-images.githubusercontent.com/185025/145608648-178e9ad5-4fa1-42ae-9652-c4a747f68879.png)

I think this is rather related to https://github.com/apache/sling-org-apache-sling-jcr-contentloader/commit/235a16eaa8cb6ed0fde56b1966e6506d1edae46f

Reply:
AEMaaCS ships with the old version 2.3.0 which still relies on admin sessions (https://issues.apache.org/jira/browse/SLING-9735) and hence does not have any service user configured for jcr.contentloader.